WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -45,8 +61,17 @@ final class BlogDashboardViewController: UIViewController {
         var snapshot = Snapshot()
         snapshot.appendSections(Section.allCases)
         snapshot.appendItems(quickLinks, toSection: Section.quickLinks)
+        snapshot.appendItems(posts, toSection: Section.posts)
         dataSource.apply(snapshot, animatingDifferences: false)
     }
+
+    private func addHeightObservers() {
+        NotificationCenter.default.addObserver(self, selector: #selector(self.methodOfReceivedNotification(notification:)), name: .postCardTableViewSizeChanged, object: nil)
+    }
+
+    @objc private func methodOfReceivedNotification(notification: Notification) {
```

Comment:
What do you think calling this method something like `updateCollectionViewHeight` instead?

Reply:
Ops! Done in e1a47cec82573f2d5247a49b3811145f85bb608f